package.json

```diff
@@ -13,14 +13,13 @@
 		"build": "dotenv -- turbo build",
 		"check": "dotenv -- turbo check:lint check:type check:format type:tests",
 		"check:format": "prettier . --check --ignore-unknown",
-		"check:lint": "dotenv -- turbo check:lint",
+		"check:lint": "node ensure-turbo-build-output.mjs && dotenv -- turbo check:lint",
 		"check:type": "dotenv -- turbo check:type type:tests",
 		"dev": "dotenv -- turbo dev",
 		"fix": "pnpm run prettify && dotenv -- turbo check:lint -- --fix",
 		"prettify": "prettier . --write --ignore-unknown",
-		"test": "dotenv -- turbo test --concurrency=1",
-		"test:ci": "dotenv -- turbo test:ci --filter=!local-mode-tests --filter=!playground-preview-worker --filter=!no-bundle-import --filter=!pages-functions-app --filter=!pages-plugin-mounted-on-root-app --filter=!workers.new --concurrency=1",
-		"test:ci:quarantine": "dotenv -- turbo test:ci --filter=local-mode-tests --filter=pages-functions-app --filter=pages-plugin-mounted-on-root-app --filter=playground-preview-worker --filter=no-bundle-import --filter=workers.new --concurrency=1",
+		"test": "dotenv -- turbo test",
+		"test:ci": "vitest run && dotenv -- turbo test:ci --filter=wrangler --filter=miniflare",
```

Comment:
Are we intentionally no longer running tests for anything else?

Reply:
Yes, everything else runs in `vitest`